package.json

```diff
@@ -61,7 +61,7 @@
     "supertest": "3.1.0"
   },
   "volta": {
-    "node": "10.21.0",
+    "node": "^16",
```

Comment:
Oh, forgot we have this volta configs in some of our repos. Keep doin what you're doin here - updating them if you see them.